meta-luneui/recipes-webos/pmloglib/pmloglib.bb

```diff
@@ -24,7 +24,7 @@ SYSTEMD_SERVICE_${PN} = "pm-log-daemon.service"
 PACKAGECONFIG ??= ""
 PACKAGECONFIG[whitelist] = "-DENABLE_WHITELIST:BOOL=TRUE, -DENABLE_WHITELIST:BOOL=FALSE"
 
-SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE};branch=webosose"
+SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE};branch=webOS-ports/webOS-OSE"
```

Comment:
instead of setting ;branch=something after ${WEBOS_PORTS_GIT_REPO_COMPLETE} you should just set WEBOS_GIT_PARAM_BRANCH = "something"